### PR TITLE
Add super pipeline sample

### PR DESCRIPTION
**Pull request by revit13**
## Why are these changes needed?

This PR adds super pipeline sample including a new kfp component.

### CODE REVIEW

**Review by blublinsky — APPROVED** [review]
LGTM

File: kfp/doc/multi_transform_pipeline.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+In the list of its input parameters, we also see `data_s3_config`. Now, we have `input_folder = path/to/B_input` (the output folder of the previous transformation pipeline) and `output_folder=/path/to/output`, the desired output folder for the whole task. if we want to execute several transformation pipelines, we have to define more intermediate folders.
+
+
+![param list](param_list2.png)
+
+
+**Note** An example super pipeline that combines several transforms, `doc_id`, `ededup`, and `fdedup`, can be found in [superworkflow_dedups_sample_wf.py](../transform_workflows/superworkflows/superworkflow_dedups_sample_wf.py).
+
+![super pipeline](super_pipeline.png)
```
Comment (roytman) [inline kfp/doc/multi_transform_pipeline.md:29]: I think the page is missing

File: kfp/transform_workflows/superworkflows/Makefile [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# Define the root of the local git clone for the common rules to be able
+# know where they are running from.
+REPOROOT=../../..
+
```
Comment (roytman) [inline kfp/transform_workflows/superworkflows/Makefile:16]: should we add `include ../../.make.transforms_workflows` ?

File: kfp/doc/multi_transform_pipeline.md (hunk 1 above)
Comment (revit13) [reply kfp/doc/multi_transform_pipeline.md:29]: Fixed. Thanks

**Review by roytman — APPROVED** [review]
LGTM

**Pull request merged**